datadriven/src/sgpp/datadriven/datamining/builder/DensityRatioEstimationMinerFactory.hpp

```diff
@@ -0,0 +1,79 @@
+// Copyright (C) 2008-today The SG++ project
+// This file is part of the SG++ project. For conditions of distribution and
+// use, please see the copyright notice provided with SG++ or at
+// sgpp.sparsegrids.org
+
+#pragma once
+
+#include <sgpp/datadriven/datamining/builder/MinerFactory.hpp>
+#include <sgpp/datadriven/datamining/configuration/DataMiningConfigParser.hpp>
+#include <sgpp/datadriven/datamining/modules/dataSource/DataSource.hpp>
+#include <sgpp/datadriven/datamining/modules/scoring/Scorer.hpp>
+#include <sgpp/datadriven/datamining/modules/dataSource/DataSourceSplitting.hpp>
+#include <sgpp/datadriven/datamining/modules/dataSource/DataSourceCrossValidation.hpp>
+#include <sgpp/datadriven/datamining/modules/fitting/ModelFittingBase.hpp>
+
+#include <string>
+#include <vector>
+
+namespace sgpp {
+namespace datadriven {
+
+/**
+ * Concrete Factory that builds an instance of #sgpp::datadriven::SparseGridMiner for Least Squares
```

Comment:
Is this the correct description for this class?

Reply:
Oups! :D I'll get to fixing those. I might fix a few more along the way, as I'm pretty sure I'm not the only one who messed these up. Thanks for noticing it.